src/fmu/settings/_resources/changelog_manager.py

```diff
@@ -25,3 +31,51 @@ def __init__(self: Self, fmu_dir: FMUDirectoryBase) -> None:
     def relative_path(self: Self) -> Path:
         """Returns the relative path to the log file."""
         return Path("logs") / LogFileName.changelog
+
+    def log_updates_to_changelog(
+        self: Self,
```

Comment:
```suggestion
    def log_update_to_changelog(
        self: Self,
```

Maybe plural, maybe `add_changelog_entry`